Packs/FiltersAndTransformers/Scripts/IfElif/README.md

```diff
@@ -0,0 +1,98 @@
+| not | Reverse the result, returns False if the result is true | not(x < 5 and x < 10) |
+
+**Regular expressions** are implemented with the "regex_match" function, in the format: `regex_match('pattern', 'string')`. The behavior of the function is controlled with the *flags* argument.
+
+**Literal strings** should preferably be surrounded by single quotes. Do _not_ use `#{...}` in a string, instead, use the `+` operator. For example: `'first ' + #{second.string} + ' third'` will be equal to the common `"first ${second.string} third"`. (This method can be used for lists too.)
+
+The following flags can be used in the *flags**argument to control the transformer's behavior:
+|Flag|Effect|Example|
+|---|---|---|
+| case_insensitive | Comparisons between strings and regex matches are case-insensitive. | `'WoRd' == 'wOrD'` |
+| list_compare | Comparing an object with a list also compares the object with all values in the list and evaluates to true if *any* comparison is true.<br>Works on operators: `<` `>` `<=` `>=` `in` `not in` `+` | `'word' in ['word1', 'word2']` |
+| regex_dot_all | Make the `.` special character match any character at all, including a newline. Without this flag, `.` will match anything *except* a newline. | `regex_match('a.b', 'a\nb')` |
+| regex_multiline | The patterns `^` and `$` will match the beginning and end of each line respectively as opposed to the beginning and end of the *string*. | `regex_match('^\d$', '1\n2\n')` |
+| regex_full_match | Regex patterns will be compared with the whole string to find a match. | `not regex_match('\d+', 'a12345')` |
+
+### Example:
+---
+##### value (Get):
+```
+${.}
+```
+
+##### conditions:
+```json
+[
+  {
+    "condition": "'www.' + #{domain.name} + '.com' not in #{approved.sites}",
+    "return": #{domain.name}
+  },
+  {
+    "condition": "#{number} >= 5 and #{path.to.string} == 'Yes'",
+    "return": "valid"
+  },
+  {
+    "condition": "regex_match('\d+', #{some.value})",
+    "return": #{value.to.return}
+  },
+  {
+    "default": #{default.value}
+  }
+]
+```
+
+##### flags:
+```
+case_insensitive,regex_dot_all,regex_multiline
+```
+
+
+## Script Data
+
+---
+
+| **Name** | **Description** |
+| --- | --- |
+| Script Type | python3 |
+| Tags | transformer, general |
+| Cortex XSOAR Version | 6.9.0 |
+
+## Inputs
+
+---
+
+| **Argument Name** | **Description** |
+| --- | --- |
+| value | The object from which to grab values, for the full context use "$\{.\}". |
+| conditions | A JSON formatted list, where all but the last items are dictionaries with the keys "condition" (holding a boolean expression) and "return" (holding the value to return if "condition" is true).<br/>The last dictionary can have the key "default" which can hold any valid JSON object to return if no "condition" was true. |
+| flags | Flags to control comparison and regular expression behavior. Possible values are: case_insensitive, regex_dot_all, regex_multiline, regex_full_match. |
```

Comment:
list_compare isnt in possible values. make sure this list is updated

Reply:
done